tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
-    "lib": ["es5", "es6", "es7", "es2017", "dom"],
-    "rootDirs": ["src"],
-    "baseUrl": "src",
```

Comment:
Can we keep those? This unlocks not pre-pending loads of `../` to imports

Reply:
dang, I missed your comment. we have been using them in one case only. I'm happy if we reintroduce / harmonize across packages at some other point in time